src/billable-services/billable-service.resource.ts

```diff
@@ -99,3 +99,25 @@ export const updateBillableService = (uuid: string, payload: UpdateBillableServi
     },
   });
 };
+
+export const createCashPoint = (payload: any) => {
+  const url = `${apiBasePath}cashPoint`;
+  return openmrsFetch(url, {
+    method: 'POST',
+    body: payload,
+    headers: {
+      'Content-Type': 'application/json',
+    },
+  });
+};
+
+export const updateCashPoint = (uuid: string, payload: any) => {
+  const url = `${apiBasePath}/cashPoint/${uuid}`;
+  return openmrsFetch(url, {
+    method: 'POST',
+    body: JSON.stringify(payload),
+    headers: {
+      'Content-Type': 'application/json',
+    },
+  });
+};
```

Comment:
This way you wouldn't need two separate functions
```suggestion
export const updateCashPoint = (uuid: string | null, payload: CashPointPayload) => {
  let url = `${apiBasePath}/cashPoint`;
  if (uuid) {
    url = `${apiBasePath}/cashPoint/${uuid}`;
  }
  return openmrsFetch(url, {
    method: uuid ? 'POST' : 'PUT',
    body: payload,
    headers: {
      'Content-Type': 'application/json',
    },
  });
};
```